docs/sources/source/nlp/dialogue_state_tracking.rst

```diff
@@ -427,78 +427,81 @@ Note, during the evaluation, the model first generates predictions and writes th
 Model Improvements
 ------------------
 
-Model improvements added to get better performance results and increase model flexibility:
+Model features added to get better performance results or increase model flexibility:
 
 - data augmentation
 - system retrieval mechanism
 - ability to make schema embeddings trainable during the model training
+- adding an attention layer between the encoder and the model heads
+
+Data augmentation is done offline with `examples/nlp/dialogue_state_tracking/data/sgd/dialogue_augmentation.py <https://github.com/NVIDIA/NeMo/blob/master/examples/nlp/dialogue_state_tracking/data/sgd/dialogue_augmentation.py>`_.
+We used 10x as augmentation factor.
```

Comment:
could you please add details of how the data augmentation works and how to run it?